java/odsa-jdbc-atp/pom.xml

```diff
@@ -1,42 +1,54 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>com.oracle.dev.jdbc</groupId>
 	<artifactId>odsa-jdbc-atp</artifactId>
 	<version>1.0-SNAPSHOT</version>
 
 	<name>odsa-jdbc-atp</name>
-	<description>A simple odsa-jdbc-atp.</description>	
-	<url>https://medium.com/oracledevs/odsa-for-java-developers-introduction-to-oracle-database-service-for-azure-part-1-bff24c787055</url>
+	<description>A simple JDBC + ODSA ATP app example</description>
+	<url>
+		https://medium.com/oracledevs/odsa-for-java-developers-introduction-to-oracle-database-service-for-azure-part-1-bff24c787055</url>
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>19</maven.compiler.source>
-		<maven.compiler.target>19</maven.compiler.target>
+		<maven.compiler.source>21</maven.compiler.source>
+		<maven.compiler.target>21</maven.compiler.target>
 	</properties>
 
 	<dependencies>
+		<!-- Oracle JDBC JARs -->
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>3.8.1</version>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>23.3.0.23.09</version>
```

Comment:
Please use 23.4

Reply:
Done

Reply:
Please use 23.4: [23.4.0.24.05](https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc11/23.4.0.24.05)

Reply:
Hi Jean, it's already using it. You're looking at a past commit, I followed all your requests. To confirm, look at 
commit 3057752bd477b00f33c85984872de672ca8e627c
https://github.com/oracle-samples/oracle-db-examples/pull/228/commits/3057752bd477b00f33c85984872de672ca8e627c